### PR TITLE
Update cofee model and examples

### PR DESCRIPTION
**Pull request by tortmayr**
- Update coffee model examples to be compliant with the latest metamodel changes.
- Fix wrong nsURI in Coffee.ecore
- Regenerate model code
- Ensure that the ResourceManager registers the EPackageConfiguration (via initialize() method)

### CODE REVIEW

File: examples/com.eclipsesource.modelserver.example/src/main/java/com/eclipsesource/modelserver/example/ExampleServerLauncher.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
 	private static String WORKSPACE_ROOT = "workspace";
 	private static String ECORE_TEST_FILE = "Coffe.ecore";
 	private static String COFFEE_TEST_FILE = "SuperBrewer3000";
-	private static String COFFEE_XMI_TEST_FILE = COFFEE_TEST_FILE + ".xmi";
+	private static String COFFEE_COFFEE_TEST_FILE = COFFEE_TEST_FILE + ".coffee";
```
Comment (edgarmueller) [inline examples/com.eclipsesource.modelserver.example/src/main/java/com/eclipsesource/modelserver/example/ExampleServerLauncher.java]: That name is a bit weird :) Maybe we can rename `COFFEE_TEST_FILE` to something else in order to have this fixed?

**Review by edgarmueller — APPROVED** [review]
LGTM, I've left one minor comment about naming.

**Comment by edgarmueller** [comment]
@tortmayr Build is failing

**Comment by tortmayr** [comment]
Thanks! I Fixed the build and resolved the naming issue.

**Pull request merged**